sky/global_user_state.py

```diff
@@ -594,19 +609,23 @@ def get_cluster_from_name(
             'storage_mounts_metadata':
                 _load_storage_mounts_metadata(storage_mounts_metadata),
             'cluster_ever_up': bool(cluster_ever_up),
+            'status_updated_at': status_updated_at,
         }
         return record
     return None
 
 
 def get_clusters() -> List[Dict[str, Any]]:
     rows = _DB.cursor.execute(
-        'select * from clusters order by launched_at desc').fetchall()
+        'select name, launched_at, handle, last_use, status, autostop, '
+        'metadata, to_down, owner, cluster_hash, storage_mounts_metadata, '
+        'cluster_ever_up, status_updated_at from clusters '
+        'order by launched_at desc').fetchall()
```

Comment:
Any reason to move away from `*`?

Reply:
See also #4211, same class of issues.

Reply:
I see. Is backward compatibility still preserved? E.g., if a cluster was launched before this PR and state.db doesn't contain `status_updated_at`, but on upgrading to this branch this line tries to select `status_updated_at`, will that work?

(I think it should  still work because `create_table` is called at module initialization, but just want to double check).

Reply:
Yes, I believe that should be fine. In fact even with `select *`, this method would crash if `status_updated_at` was missing because we would not have enough columns to unpack.